src/shared/components/Dialog/Dialog.styles.ts

```diff
@@ -52,21 +47,17 @@ export const DialogContainer = styled.div<SizeProps>`
 
 const headerDividersStyles = css`
   box-shadow: inset 0 -1px 0 0 ${colors.gray[600]};
+  padding-bottom: var(--local-size-dialog-padding);
 `
 
-const headerDenseStyles = css`
-  padding-bottom: ${sizes(3)};
-`
-
-export const Header = styled.div<HeaderProps>`
+export const Header = styled.div<DividersProps>`
   display: flex;
   flex-direction: row;
   justify-content: space-between;
   align-items: flex-start;
-  padding: var(--local-size-dialog-padding);
+  padding: var(--local-size-dialog-padding) var(--local-size-dialog-padding) 0;
```

Comment:
according to the design on mobile padding is a little different when the title is present. It's 24px top and bottom not 16px. 
Yours: 
![image](https://user-images.githubusercontent.com/51168865/140480319-8b414f2b-526b-4e81-b5ce-c1a1fe96f6fe.png)
Design: 
![image](https://user-images.githubusercontent.com/51168865/140480394-a118bf92-cc2e-4658-8067-8b90d30f8ba4.png)

Reply:
Also, I think the header should have bottom padding if it's possible, not the dialog content. Look at the screen:
![image](https://user-images.githubusercontent.com/51168865/140481011-4f66ea62-1d63-4464-ac28-c6c55f73b420.png)

(I know that it's an extreme situation, but maybe we could consider adding it if it's not problematic?)

Reply:
So regarding the first issue you brought up, I think the padding actually is 16px, you need to look at the biggest element in the header which is the close button:
<img width="595" alt="Screenshot 2021-11-05 at 10 23 21" src="https://user-images.githubusercontent.com/12646744/140488253-c6705a0a-59d7-496c-a26f-e5268d7e2d10.png">

Regarding the second case for when we're scrolling I'm afraid you are right but I just don't want to fix it :P Adding the padding to the content instead of the header fixed the scrollbar issue I've mentioned. Also, in theory, whenever scroll will be needed inside the container, the designers should use the `dividers` variant so we shouldn't run into this case at all

Reply:
Regarding the first one - my mistake, you're right. Regarding the second point - we can ignore this.